action-server/actions/__main__.py

```diff
@@ -0,0 +1,18 @@
+from actions.answers import QaResponse, QaProtocol
+import os, sys
+
+
+FAQ_URL_ENV_KEY = "faq_url"
+DEFAULT_FAQ_URL = "https://covidfaq.dialoguecorp.com"
+
+
+def main(question: str = ""):
```

Comment:
I don't think an empty string is a sensible default for this function.  It should be removed.